src/internal/pachsql/db.go

```diff
@@ -46,6 +46,9 @@ type Tx = sqlx.Tx
 // Stmt is an alias for sqlx.Stmt which is the standard prepared statement type used throught the project
 type Stmt = sqlx.Stmt
 
+// ID is the auto-incrementing primary key used for entries in postgres tables.
+type ID uint64
```

Comment:
Let's not include this here.  This doesn't give us any more type safety than just a uint64, since everything would depend on this, and then there is no distinction between the identifiers for different objects.

It would be a good idea to create types `RepoID` `CommitID` all based on uint64, because then we would have type safety.  It would be harder to confuse the identifier for a commit for that of a repo.